hubgrep/models/hosting_service.py

```diff
@@ -20,11 +20,12 @@ class HostingService(db.Model):
 
     api_url = db.Column(db.String(500), unique=True, nullable=False)
    
-    # frontend label
-    label = db.Column(db.String(80))
+    export_timestamp = db.Column(db.DateTime)
+    repo_count = db.Column(db.Integer)
 
-    def set_service_label(self):
-        self.label = re.split("//", self.landingpage_url)[1].rstrip("/")
+    @property
+    def domain(self):
+        return re.split("//", self.landingpage_url)[1].rstrip("/")
```

Comment:
made this dynamic. 
we had a bug before somewhere else (i think that would have been fixed now as well :roll_eyes: ), causing the label to be the api domain.
anyways, since we have the object with all values anyways, we can just build that on the fly.

Reply:
todo: store this again, we want to filter on this again